CHANGELOG.md

```diff
@@ -3,44 +3,45 @@ All notable changes to this project are documented in this file.
 
 ## Unreleased
 
+## [0.3.2] - 2022-11-28
+
 ### Changed
 
  - When using a two way binding `foo <=> bar`, the default value will always be the one of `bar`.
-   There was a warning about this change in previous versions (#1394)
- - `Window`'s `default-font-size` property is now always set to a non-zero value, provided by
-   either the style or the backend.
- - In the interpreter, calling `set_property` or `get_property` on properties of the base no longer works.
- - Renamed the `Keys` namespace for use in `key-pressed`/`key-released` callbacks to `Key`. The
-   old name continues to work.
+   There was a warning about this change in previous versions (#1394).
  - Disallow overrides or duplicated declarations of callbacks. Previously they were silently overwritten,
    now an error is produced.
- - The name "init" is now a reserved name in callbacks and properties.
+ - The name `init` is now a reserved name in callbacks and properties.
+ - In the interpreter, calling `set_property` or `get_property` on properties of the base returns an error.
  - Deprecated `slint::WindowEvent` and `slint::PointerEventButton` and moved them to the `slint::platform` module.
+ - Renamed the `Keys` namespace for use in `key-pressed`/`key-released` callbacks to `Key`. The
+   old name continues to work.
+ - The style or the backend now always set a value for the `Window`'s `default-font-size` property.
 
 ### Added
 
- - Added `material` style with `material-light` and `fluent-dark` as explicit styles
- - Added `Window::is_visible`
+ - Added new `material` style based on Material Design, with `material-light` and `material-dark` as variants.
+ - Added `Window::is_visible` in Rust and C++.
  - Added `From<char>` for `SharedString` in Rust.
  - Added `KeyPressed` and `KeyReleased` variants to `slint::WindowEvent` in Rust, along
    with `slint::platform::Key`, for use by custom platform backends.
  - Added support for the implicitly declared `init` callback that can be used to run code when
    an element or component is instantiated.
  - Properties can be annotated with `in`, `out`, `in-out`, or `private`.
- - Transitions can now be declared directly within the state
- - Online editor: The property view can now edit properties
- - LSP preview: highlight the selected element in the preview
- - LSP: Added a setting to change the style and the include paths
+ - Transitions can now be declared directly within the state.
+ - Online editor: The property view can now edit properties.
```

Comment:
Personally, I think we should list changes in the online editor in a different solution - ideally IMO in an "About - Changes" menu in the online editor editor. I think here it just adds noise.

Reply:
I agree we could have different section for the different part.
But right now it's just one line, i wouldn't consider that as noise.